src/services/keystroke.ts

```diff
@@ -513,8 +551,9 @@ class Controller_keystroke extends Controller_focusBlur {
     var selection = cursor.selection;
     if (selection) {
       cursor.controller.aria
+        // clearing first because selection fires several times, and we don't want repeated speech.
         .clear()
-        .queue(selection.join('mathspeak', ' ').trim() + ' selected'); // clearing first because selection fires several times, and we don't want repeated speech.
+        .queueSelected(selection.join('mathspeak', ' ').trim());
```

Comment:
There's another occurrence of `selection.join('mathspeak', ' ').trim() + ' selected'` on line 212—should that change as well?

Reply:
Thank you!